src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
+    padding: 0.5rem;
+  }
+
+  &__toggle {
+    background: none;
+    border: 0;
+    transition: transform 0.25s ease-in-out;
```

Comment:
Please use the Vanilla animation mixin: https://docs.vanillaframework.io/settings/animation-settings/

Reply:
I did try but hit pain - issue raised here: https://github.com/canonical-web-and-design/vanilla-framework/issues/2576